axis_backend/apps/audit/serializers/entity_change_serializer.py

```
@@ -0,0 +1,36 @@
+    """Comprehensive serializer for entity change details."""
+
+    field_changes_count = serializers.SerializerMethodField()
+
+    class Meta:
+        model = EntityChange
+        fields = [
+            'id', 'entity_type', 'entity_id', 'change_type',
+            'changed_at', 'changed_by', 'change_reason',
+            'old_data', 'new_data', 'metadata', 'is_active',
+            'field_changes_count', 'created_at', 'updated_at', 'deleted_at'
+        ]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)